generators/list-versions/index.js

```diff
@@ -0,0 +1,36 @@
+/* eslint no-underscore-dangle: 0 */
+
+/*
+* License-Identifier: Apache-2.0
+*/
+const Generator = require('yeoman-generator');
+
+const config = require('../config');
+const repositoryUtils = require('../repositoryUtils');
+
+module.exports = class extends Generator {
+  async printAllVersions() {
+    const allVersions = await repositoryUtils.getAvailableTags();
+
+    const versionsSortedAndMarked = repositoryUtils
+      .sortVersions(allVersions)
+      .map(this._markAsCurrent)
+      .map(this._markAsCompatible);
+
+    this.log(JSON.stringify(versionsSortedAndMarked, null, 2));
```

Comment:
tego nie czaję - niby wypluwamy JSON, który jest ustruktyzowany, może być dalej przetwarzany itd. ale w tym JSONie nie mamy wartości do użycia przez inne skrypty, tylko stringi sztucznie obrobione tak, żeby łatwo je było przeczytać użytkownikowi.

imho tutaj powinniśmy wypluć przyjazny dla użytkownika output, najprościej pewnie listę wersji w osobnych linijkach, ew. oznaczone jako current i compatible (co to w ogóle znaczy compatible?)